README.md
# NAME

Map::Metro::Plugin::Map::Budapest - Map::Metro map for Budapest

<div>
    <p>
    <img src="https://img.shields.io/badge/perl-5.16+-blue.svg" alt="Requires Perl 5.16+" />
    <a href="https://travis-ci.org/Csson/p5-Map-Metro-Budapest"><img src="https://api.travis-ci.org/Csson/p5-Map-Metro-Budapest.svg?branch=master" alt="Travis status" /></a>
    <a href="http://cpants.cpanauthors.org/release/CSSON/Map-Metro-Plugin-Map-Budapest-0.1002"><img src="http://badgedepot.code301.com/badge/kwalitee/CSSON/Map-Metro-Plugin-Map-Budapest/0.1002" alt="Distribution kwalitee" /></a>
    <a href="http://matrix.cpantesters.org/?dist=Map-Metro-Plugin-Map-Budapest%200.1002"><img src="http://badgedepot.code301.com/badge/cpantesters/Map-Metro-Plugin-Map-Budapest/0.1002" alt="CPAN Testers result" /></a>
    <img src="https://img.shields.io/badge/coverage-90.0%-yellow.svg" alt="coverage 90.0%" />
    </p>
</div>

# VERSION

Version 0.1002, released 2019-02-27.

# SYNOPSIS

    use Map::Metro;
    my $graph = Map::Metro->new('Budapest')->parse;

Or

        $ map-metro.pl route Budapest "II. János Pál pápa tér" "Opera"

# DESCRIPTION

See [Map::Metro](https://metacpan.org/pod/Map::Metro) for usage information.

# Status

[Map::Metro::Plugin::Map::Budapest::Lines](https://metacpan.org/pod/Map::Metro::Plugin::Map::Budapest::Lines)

This map consists of:

- The four metro lines \[[wikipedia](https://en.wikipedia.org/wiki/Budapest_Metro)\]

# SOURCE

[https://github.com/Csson/p5-Map-Metro-Budapest](https://github.com/Csson/p5-Map-Metro-Budapest)

# HOMEPAGE

[https://metacpan.org/release/Map-Metro-Plugin-Map-Budapest](https://metacpan.org/release/Map-Metro-Plugin-Map-Budapest)

# AUTHOR

Erik Carlsson <info@code301.com>

# COPYRIGHT AND LICENSE

This software is copyright (c) 2016 by Erik Carlsson.

This is free software; you can redistribute it and/or modify it under
the same terms as the Perl 5 programming language system itself.


Changes
Revision history for Map-Metro-Plugin-Map-Budapest

{{$NEXT}}
 [API Changes]

 [Bug Fixes]

 [Documentation]

 [Enhancements]

 [New Features]

 [Other Changes]

 [Requirements]

 [Stats & Dependency Changes]

 [Testing]

0.1002 2019-02-27T22:12:40Z UTC
 [Other Changes]
 - Rebuilt with current dzil configuration.

 [Stats & Dependency Changes]
 - 3 files changed, 11 insertions(+), 26 deletions(-)
 - (run req) + Moose 2.1400
 - (run req) + namespace::autoclean 0.24
 - (run req) ~ Map::Metro 0.1900 --> 0.2300
 - (run req) - Moose::Role
 - (test req) + IO::Handle (any)
 - (test req) + IPC::Open3 (any)
 - (conf req) + ExtUtils::MakeMaker (any)
 - (conf req) + File::ShareDir::Install 0.06
 - (conf req) - Module::Build::Tiny
 - (dev req) + Badge::Depot 0.0103
 - (dev req) + Badge::Depot::Plugin::Coverage (any)
 - (dev req) + Badge::Depot::Plugin::Cpantesters (any)
 - (dev req) + Badge::Depot::Plugin::Kwalitee (any)
 - (dev req) + Badge::Depot::Plugin::Perl (any)
 - (dev req) + Badge::Depot::Plugin::Travis (any)
 - (dev req) + Dist::Zilla 6.012
 - (dev req) + Dist::Zilla::Plugin::Authority 1.009
 - (dev req) + Dist::Zilla::Plugin::BumpVersionAfterRelease::Transitional
   0.008
 - (dev req) + Dist::Zilla::Plugin::ChangeStats::Dependencies::Git 0.0200
 - (dev req) + Dist::Zilla::Plugin::Clean 0.07
 - (dev req) + Dist::Zilla::Plugin::DistIller::MetaGeneratedBy (any)
 - (dev req) + Dist::Zilla::Plugin::Git 2.045
 - (dev req) + Dist::Zilla::Plugin::Git::CheckFor::CorrectBranch 0.014
 - (dev req) + Dist::Zilla::Plugin::Git::Commit (any)
 - (dev req) + Dist::Zilla::Plugin::Git::Contributors 0.034
 - (dev req) + Dist::Zilla::Plugin::InstallRelease 0.008
 - (dev req) + Dist::Zilla::Plugin::MakeMaker (any)
 - (dev req) + Dist::Zilla::Plugin::MetaConfig (any)
 - (dev req) + Dist::Zilla::Plugin::NextRelease::Grouped 0.0200
 - (dev req) + Dist::Zilla::Plugin::PromptIfStale 0.055
 - (dev req) + Dist::Zilla::Plugin::RewriteVersion::Transitional 0.007
 - (dev req) + Dist::Zilla::Plugin::Test::Compile 2.057
 - (dev req) + Dist::Zilla::Plugin::Test::EOF 0.0501
 - (dev req) + Dist::Zilla::Plugin::Test::EOL 0.18
 - (dev req) + Dist::Zilla::Plugin::Test::Kwalitee::Extra v0.2.1
 - (dev req) + Dist::Zilla::Plugin::Test::NoTabs 0.15
 - (dev req) + Dist::Zilla::Plugin::Test::Version 1.05
 - (dev req) + Pod::Weaver::Section::GenerateSection 1.01
 - (dev req) + Test::EOF (any)
 - (dev req) + Test::EOL (any)
 - (dev req) + Test::More 0.88
 - (dev req) + Test::NoTabs 1.4
 - (dev req) + Test::Version 1
 - (dev req) ~ Dist::Zilla::Plugin::CheckChangesHasContent (any) --> 0.008
 - (dev req) ~ Dist::Zilla::Plugin::CopyFilesFromBuild (any) --> 0.170880
 - (dev req) ~ Dist::Zilla::Plugin::GithubMeta (any) --> 0.54
 - (dev req) ~ Dist::Zilla::Plugin::MetaProvides::Package (any) -->
   2.004003
 - (dev req) ~ Dist::Zilla::Plugin::PodWeaver (any) --> 4.006
 - (dev req) ~ Dist::Zilla::Plugin::ReadmeAnyFromPod (any) --> 0.163250
 - (dev req) ~ Dist::Zilla::Plugin::RunExtraTests (any) --> 0.028
 - (dev req) ~ Dist::Zilla::Plugin::Test::ReportPrereqs (any) --> 0.027
 - (dev req) ~ Pod::Weaver::Section::Badges (any) --> 0.0401
 - (dev req) ~ Test::Warnings (any) --> 0.026
 - (dev req) - Dist::Zilla::Plugin::CheckExtraTests
 - (dev req) - Dist::Zilla::Plugin::ContributorsFromGit
 - (dev req) - Dist::Zilla::Plugin::MetaProvides::Class
 - (dev req) - Dist::Zilla::Plugin::ModuleBuildTiny
 - (dev req) - Dist::Zilla::Plugin::NextRelease
 - (dev req) - Dist::Zilla::Plugin::NextVersion::Semantic
 - (dev req) - Dist::Zilla::Plugin::OurPkgVersion
 - (dev req) - Dist::Zilla::Plugin::PodnameFromClassname
 - (dev req) - Dist::Zilla::Plugin::Prereqs::Plugins
 - (dev req) - Dist::Zilla::Plugin::PreviousVersion::Changelog
 - (dev req) - Dist::Zilla::Plugin::ReversionOnRelease
 - (dev req) - Pod::Weaver::Section::Homepage::DefaultCPAN
 - (dev req) - Pod::Weaver::Section::Source::DefaultGitHub
 - (dev sug) + Dist::Iller 0.1408
 - (dev sug) + Dist::Iller::Config::Author::CSSON 0.0322
 - (dev sug) + Dist::Iller::Config::Author::CSSON::MapMetro 0.0322

0.1001 2015-03-23T17:35:40Z CET
 [Enhancements]
 - Remove 'map_version' method. No longer used.

 [Meta]
 - iller.yaml replaces iller.ini (generates dist/weaver.ini).
 - Added Travis-ci integration.

 [Requirements]
 - Changed required Perl version from 5.14 to 5.16 since that is what
   Map::Metro requires.

0.1000 2015-01-08T21:45:29Z CET
 - Initial release


lib/Map/Metro/Plugin/Map/Budapest.pm
use 5.14.0;

package Map::Metro::Plugin::Map::Budapest;

# ABSTRACT: Map::Metro map for Budapest
# AUTHORITY
our $VERSION = '0.1003';

use Moose;
with 'Map::Metro::Plugin::Map';

has '+mapfile' => (
    default => 'map-budapest.metro',
);
sub map_package {
    return __PACKAGE__;
}

1;

__END__

=pod

=encoding utf-8

=head1 SYNOPSIS

    use Map::Metro;
    my $graph = Map::Metro->new('Budapest')->parse;

Or

	$ map-metro.pl route Budapest "II. János Pál pápa tér" "Opera"

=head1 DESCRIPTION

See L<Map::Metro> for usage information.

=head1 Status

L<Map::Metro::Plugin::Map::Budapest::Lines>

This map consists of:

=for :list
* The four metro lines [L<wikipedia|https://en.wikipedia.org/wiki/Budapest_Metro>]

=for HTML <p><a href="https://raw.githubusercontent.com/Csson/p5-Map-Metro-Budapest/master/static/images/budapest.png"><img src="https://raw.githubusercontent.com/Csson/p5-Map-Metro-Budapest/master/static/images/budapest.png" style="max-width: 600px" /></a></p>

=cut


Makefile.PL
# This file was automatically generated by Dist::Zilla::Plugin::MakeMaker v6.012.
use strict;
use warnings;

use 5.016000;

use ExtUtils::MakeMaker;

use File::ShareDir::Install;
$File::ShareDir::Install::INCLUDE_DOTFILES = 1;
$File::ShareDir::Install::INCLUDE_DOTDIRS = 1;
install_share dist => "share";


my %WriteMakefileArgs = (
  "ABSTRACT" => "Map::Metro map for Budapest",
  "AUTHOR" => "Erik Carlsson <info\@code301.com>",
  "CONFIGURE_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0,
    "File::ShareDir::Install" => "0.06"
  },
  "DISTNAME" => "Map-Metro-Plugin-Map-Budapest",
  "LICENSE" => "perl",
  "MIN_PERL_VERSION" => "5.016000",
  "NAME" => "Map::Metro::Plugin::Map::Budapest",
  "PREREQ_PM" => {
    "Map::Metro" => "0.2300",
    "Moose" => "2.1400",
    "namespace::autoclean" => "0.24"
  },
  "TEST_REQUIRES" => {
    "ExtUtils::MakeMaker" => 0,
    "File::Spec" => 0,
    "IO::Handle" => 0,
    "IPC::Open3" => 0,
    "Test::More" => "0.96"
  },
  "VERSION" => "0.1003",
  "test" => {
    "TESTS" => "t/*.t"
  }
);


my %FallbackPrereqs = (
  "ExtUtils::MakeMaker" => 0,
  "File::Spec" => 0,
  "IO::Handle" => 0,
  "IPC::Open3" => 0,
  "Map::Metro" => "0.2300",
  "Moose" => "2.1400",
  "Test::More" => "0.96",
  "namespace::autoclean" => "0.24"
);


unless ( eval { ExtUtils::MakeMaker->VERSION(6.63_03) } ) {
  delete $WriteMakefileArgs{TEST_REQUIRES};
  delete $WriteMakefileArgs{BUILD_REQUIRES};
  $WriteMakefileArgs{PREREQ_PM} = \%FallbackPrereqs;
}

delete $WriteMakefileArgs{CONFIGURE_REQUIRES}
  unless eval { ExtUtils::MakeMaker->VERSION(6.52) };

WriteMakefile(%WriteMakefileArgs);

{
package
MY;
use File::ShareDir::Install qw(postamble);
}
